File: tools/dbench/answers/python_haiku/s1/m1_oldest.py
with open("users.txt") as f:
    lines = f.readlines()
oldest = None
max_age = -1
for line in lines:
    parts = line.strip().split()
    if len(parts) == 3:
        try:
            age = int(parts[2])
            if age > max_age:
                max_age = age
                oldest = parts[1]
        except ValueError:
            pass
if oldest:
    print(f"{oldest} {max_age}")

File: tools/dbench/answers/rust_haiku/s2/m1_oldest.rs
use std::fs;

fn main() {
    let content = fs::read_to_string("users.txt").unwrap_or_default();
    let mut max_age = -1;
    let mut max_name = "";

    for line in content.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i32>() {
                if age > max_age {
                    max_age = age;
                    max_name = parts[1];
                }
            }
        }
    }

    println!("{} {}", max_name, max_age);
}
